services/update.py

```diff
@@ -243,47 +190,62 @@ def get_update(self):
 
         return False
 
-    def get_bad_rdf(self):
-        return bad_rdf
-
-    def get_rdf(self):
+    def get_output(self):
```

Comment:
https://github.com/mozilla/addons-server/pull/8175/files#diff-a3aebfacc7f05d955e54b62d722ae70aL305 still references `get_rdf`

Reply:
is an integration test possible to catch that kind of mistake?

Reply:
doh! I can probably try to call `application()` directly